server/routes/api/browser.py

```diff
@@ -33,4 +35,29 @@ def triple_api(dcid):
 @bp.route('/popobs/<path:dcid>')
 def popobs_api(dcid):
     """Returns all the triples given a node dcid."""
-    return dc.get_pop_obs(dcid)
\ No newline at end of file
+    return dc.get_pop_obs(dcid)
+
+
+@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@bp.route('/propvals/<path:prop>/<path:dcid>')
+def get_property_value(dcid, prop):
+    response = fetch_data('/node/property-values', {
+        'dcids': [dcid],
+        'property': prop,
+    },
+                          compress=False,
+                          post=False)
+    result = {}
+    result["property"] = prop
```

Comment:
the caller should know "prop", is result["property"] necessary?

Reply:
Also can you see if can reuse this function? https://github.com/datacommonsorg/website/blob/99bc65c31ea19c1d72b8b8a1fa667941a3dedf8a/server/routes/api/place.py#L63-L65

If so, can make a shared library to host all these common apis

Reply:
I don't think result["property"] is necessary, but I added it so that when the client side gets multiple property value promises finishing, it's more clear which property each result of values is referring to. 

That function isn't an api endpoint. Do you mean to move that function to a shared library and then in the shared library, make an api endpoint that calls the function?

Reply:
then keep "property" in result sounds good. I looked back and the way you use fetch_data() is better, so let's keep this and consolidate them in later PR